tests/test_tools.py

```diff
@@ -2015,17 +2015,15 @@ def test_filter_employment_for_advisees(inp, expected):
 person2 = {"_id":"nromanov", "aka":"black widow", "name":"natasha romanov"}
 PEOPLE = [person1, person2]
 
-presentation1 = {"_id":"abc", "authors":["tstark"], "begin_year":2018, "begin_month":1, "begin_day":1,
-                 "department":"apam", "institution":"columbiau", "status":"accepted", "type":"award"}
-presentation2 = {"_id":"ghi", "authors":["tstark","nromanov"], "begin_year":2019, "begin_month":1, "begin_day":2,
-                 "end_year":2019, "end_month":1, "end_day":8, "department":"physics", "institution":"rutgersu",
-                 "status":"cancelled", "type":"poster"}
+presentation1 = {"_id":"abc", "authors":"tstark", "date":"2018-01-01", "department":"apam",
```

Comment:
you changed them both, so now you are not testing the day, month year pattern...... :(